src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2112,6 +2112,7 @@ because xbuild doesn't support framework reference assemblies.
       SupportedAbis="@(_BuildTargetAbis)"
       Debug="$(AndroidIncludeDebugSymbols)"
       EnableCompression="$(AndroidEnableAssemblyCompression)"
+      ProjectGuid="$(ProjectGuid)"
```

Comment:
It looks like when I build a .NET 5 project, `$(ProjectGuid)` is blank?

```
dotnet build foo.csproj -bl
```
![image](https://user-images.githubusercontent.com/840039/84159443-b03b8980-aa32-11ea-8226-f182abc9a0cf.png)

Maybe this has to use [`$(MSBuildProjectFile)`](https://docs.microsoft.com/visualstudio/msbuild/msbuild-reserved-and-well-known-properties)?